manifests/0000_50_cluster-monitoring-operator_02-role.yaml

```diff
@@ -302,14 +294,6 @@ rules:
   verbs:
   - list
   - watch
-- apiGroups:
-  - security.openshift.io
-  resourceNames:
-  - node-exporter
-  resources:
-  - securitycontextconstraints
-  verbs:
-  - use
```

Comment:
i believe this is correct, there is no need to have the `node-exporter` SCC on the CMO pod.